src/instructions.spec.ts

```diff
@@ -469,6 +469,19 @@ describe('Cortex-M0+ Instruction Set', () => {
     expect(registers.V).toEqual(false);
   });
 
+  it('should execute an `cmp	r2, r0` instruction', async () => {
+    await cpu.setPC(0x20000000);
+    await cpu.writeUint16(0x20000000, opcodeCMPregT1(r2, r0));
+    await cpu.setRegisters({ r0: 0xb71b0000 });
+    await cpu.setRegisters({ r2: 0x00b71b00 });
+    await cpu.setRegisters({ xPSR: 0x61000000 });
```

Comment:
Instead of setting xPSR directly, let's set the specific flags (like we did in the "should execute `add sp, r8` instruction and not update the flags" test case)

Reply:
Also, we can have a single call to setRegisters setting all the registers at once (instead of one call per register)